src/notification-banner/types.ts

```diff
@@ -8,6 +8,13 @@ export interface NotificationBannerProps
     sticky?: boolean | undefined;
     onDismiss?: (() => void) | undefined;
     "data-testid"?: string | undefined;
+    /**
+     * Specifies the height to be displayed outside of
+     * the collapsible zone
+     */
+    collapsedHeight?: number | undefined;
```

Comment:
since the constraint is on the content, would be good to be more explicit

```suggestion
    collapsedContentHeight?: number | undefined;
```